common/cpu/amd/raphael/igpu.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, pkgs, ... }: 
+
+{
+  # Disables scatter/gather which was introduced with kernel version 6.2
+  # It produces completely white or flashing screens when enabled while using the iGPU of Ryzen 7000-series CPUs (Raphael)
+
+  imports = [ ../. ];
+
+  boot = lib.mkMerge [
+    (lib.mkIf (lib.versionOlder pkgs.linux.version "6.1") {
+      kernelPackages = pkgs.linuxPackages_latest;
+      kernelParams = ["amdgpu.sg_display=0"];  
```

Comment:
Is the kernelParams flag not required if the kernel is new enough?

Reply:
Yep. From a LTS perspective 5.15 is too old, 6.1 is new enough and doesn't need the flag, 6.2 introduced scatter/gather which makes this flag necessary, and 6.3 still needs the flag too.